### ISSUE TITLE
[Docs] Saving and Loading Models

### ISSUE THREAD

**Issue by sahandrez** [cornellius-gp/gpytorch#677]
Hello, 

I am new to GPyTorch and I'm having trouble saving and loading models. As already mentioned [here](https://github.com/cornellius-gp/gpytorch/issues/244), currently there is no example for this. I am saving and loading the model similar to regular PyTorch models, but I am getting `size mismatch for covar_module.base_kernel.raw_lengthscale` error. I have checked [this](https://github.com/cornellius-gp/gpytorch/pull/3) out and tried saving/loading the training data for initialization of the likelihood prior, but still no success. 

Can you please give a minimal save/load procedure for ExactGP model? 

Thanks,
Sahand

**Comment by jacobrgardner** [comment]
@sahandrez  Yeah it has been on my to-do list for a long time to put together documentation for basic functionality like this. I'll get some form of this done by the end of Monday.

**Comment by sahandrez** [comment]
@jacobrgardner Thanks a lot!

**Comment by jacobrgardner** [comment]
The following works for me:
```python
state_dict = model.state_dict()
...
# Later
model.load_state_dict(state_dict)
```
Does this not work for you? If so, do you have a small code example that fails?

**Comment by sahandrez** [comment]
It's working now. I hadn't realized that I should initialize the second model with the same training set. Thanks!
